runner/plugins/plugin-jest.ts

```diff
@@ -0,0 +1,38 @@
+import { gql } from 'graphql-request'
+import { client } from '../core/api'
+import { PluginInput } from '../types'
+
+const QUERY = gql`
+  query JestConfig($owner: String!, $name: String!, $configPath: String!) {
+    repository(owner: $owner, name: $name) {
+      jestConfig: object(expression: $configPath) {
+        ... on Blob {
+          text
+        }
+      }
+    }
+  }
+`
+
+export async function jestConfig(project: PluginInput) {
+  const { owner, name, defaultBranchName } = project
+  let jestConfig
+  const { repository } = await client.request(QUERY, {
+    owner,
+    name,
+    configPath: `${defaultBranchName}:jest.config.js`,
+  })
+  jestConfig = repository.jestConfig?.text
+  if (!jestConfig) {
+    const { repository } = await client.request(QUERY, {
+      owner,
+      name,
+      configPath: `${defaultBranchName}:jest.config.base.js`,
+    })
+    jestConfig = repository.jestConfig?.text
+    if (!jestConfig) {
+      return {}
+    }
+  }
```

Comment:
I would make it a small utility function which receives `project: PluginInput` and jest config file name, something like:

```js
async function getJestConfig(project: PluginInput, fileName: string) {
  const { owner, name, defaultBranchName } = project
  const { repository } = await client.request(QUERY, {
    owner,
    name,
    configPath: `${defaultBranchName}:${fileName}`,
  })
  const jestConfig = repository.jestConfig?.text
  if (!jestConfig) {
    return undefined
  }
  return JSON.parse(jestConfig.substring('module.exports = '.length))
}

export async function jestConfig(project: PluginInput) {
  let config = await getJestConfig(project, 'jest.config.js')
  if (config) return config

  config = await getJestConfig(project, 'jest.config.base.js')
  if (config) return config

  return {}
}
```

Then in the future we can even pass `fileName` from `stats.config.js` to the plugin.